internal/ingress/controller/parser/parser.go

```diff
@@ -573,8 +603,14 @@ func overrideRoute(route *Route,
 	if len(r.Headers) != 0 {
 		route.Headers = r.Headers
 	}
+	// grpc(s) doesn't accept strip_path
 	if len(r.Protocols) != 0 {
 		route.Protocols = cloneStringPointerSlice(r.Protocols...)
+		for _, val := range r.Protocols {
+			if *val == *kong.String("grpc") || *val == *kong.String("grpcs") {
+				route.StripPath = kong.Bool(false)
```

Comment:
I think `StripPath` should be `nil` in this case, else Kong will return a `400`.